CMakeLists.txt

```diff
@@ -365,11 +365,11 @@ IF (CMAKE_COMPILER_IS_GNUCXX)
     # information about the class hierarchy graph is more complete. It is
     # recommended to first consider suggestions of -Wsuggest-final-types and
     # then rebuild with new annotations.
-    TARGET_COMPILE_OPTIONS(${CMAKE_PROJECT_NAME} PRIVATE -Wsuggest-final-methods)
+    TARGET_COMPILE_OPTIONS(${CMAKE_PROJECT_NAME} PRIVATE )#-Wsuggest-final-methods)
```

Comment:
Locale compilation specific, must not be send to the master branch. Moreover must comment the entire line (in local)